### ISSUE TITLE
[4.x] TypeError on creating new stencil

### ISSUE THREAD

**Issue by jimirobaer** [verbb/formie#859]
**Description**

On a clean install with Formie and Craft CMS 4.0.0-RC1, I can't create stencils as it throws a TypeError.

```
verbb\formie\services\Stencils::getStencilById(): Argument #1 ($id) must be of type int, null given, called in /vendor/verbb/formie/src/controllers/StencilsController.php on line 135
```

**Steps to reproduce**
1. Install Formie
2. Create a new stencil by entering any name

**Additional info**
- Plugin version: 2.0.0-beta.2
- Craft version: 4.0.0-RC1
- Multi-site: No

**Additional context**

Stack Trace

```
TypeError: verbb\formie\services\Stencils::getStencilById(): Argument #1 ($id) must be of type int, null given, called in /vendor/verbb/formie/src/controllers/StencilsController.php on line 135 and defined in /vendor/verbb/formie/src/services/Stencils.php:100
Stack trace:
#0 /vendor/verbb/formie/src/controllers/StencilsController.php(135): verbb\formie\services\Stencils->getStencilById(NULL)
#1 [internal function]: verbb\formie\controllers\StencilsController->actionSave()
#2 /vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#3 /vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#4 /vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('save', Array)
#5 /vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('formie/stencils...', Array)
#6 /vendor/craftcms/cms/src/web/Application.php(625): craft\web\Application->runAction('formie/stencils...', Array)
#7 /vendor/craftcms/cms/src/web/Application.php(280): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#8 /vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#9 /web/index.php(12): yii\base\Application->run()
#10 /.composer/vendor/laravel/valet/server.php(234): require('...')
#11 {main}
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.3](https://github.com/verbb/formie/releases/tag/2.0.0-beta.3)
